### PR TITLE
DSERV-50-2bit

### PR DESCRIPTION
**Pull request by mingjiecn**
Set default window to 50 right now. And the return reference sequence look like this:
image

### CODE REVIEW

File: genomic_data_service/search.py [hunk 1]
```diff
@@ -136,3 +142,32 @@ def region_search():
         'regions': region_service.regions,
         'regions_per_file': region_service.regions_per_file
     })
+
+def get_sequence(assembly, coordinate, window=50):
+    if assembly == 'GRCh38':
+        seq_reader = py2bit.open(TWO_BIT_HG38_FILE_PATH)
+    else:
+        seq_reader = py2bit.open(TWO_BIT_HG19_FILE_PATH)
```
Comment (ottojolanki) [inline genomic_data_service/search.py:158]:
```suggestion
    if assembly == 'GRCh38':
        seq_reader = py2bit.open(TWO_BIT_HG38_FILE_PATH)
    elif assembly == 'hg19':
        seq_reader = py2bit.open(TWO_BIT_HG19_FILE_PATH)
    else:
        raise ValueError("Assembly has to be either GRCh38 or hg19.")
```

Comment (ottojolanki) [reply genomic_data_service/search.py:158]: probably safer? could cause headache if someone tries some synonym (like using "hg38" and unexpectedly and silently getting hg19 sequence back).

**Review by ottojolanki — CHANGES_REQUESTED** [review]
One suggestion, other than that LGTM.

File: genomic_data_service/search.py (hunk 1 above)
Comment (mingjiecn) [reply genomic_data_service/search.py:158]: assembly is get from function extract_search_params(), which will only return valid assembly. Only issue right now is valid assembly includes mouse assembly which we won't use(see constant GENOME_TO_ALIAS). So we may want to remove mouse assembly at some time point.

Comment (hitz) [reply genomic_data_service/search.py:158]: there is a 3rd human assembly (CHM13/T2T) that we may move to in the future.

**Comment by mingjiecn** [comment]
Previously, for search and summary endpoint, the default assembly is GRCh37 and all the invalid assembly will be changed to GRCh37. This is misleading to user. Will not set default for assembly and validate the input assembly for query.
image
image

**Review by ottojolanki — APPROVED** [review]
LGTM

**Pull request merged**